stripe/api_resources/refund.py

```diff
@@ -54,6 +55,8 @@ def cancel(self, idempotency_key=None, **params):
         )
 
     class TestHelpers(APIResourceTestHelpers):
+        _resource_cls: Type["Refund"]
```

Comment:
Do we need to type these?

Reply:
At the end of the day, it doesn't *really* matter as this is code generated, but I do think it helps communicate what `APIResourceTestHelpers` expects from its implementors.